CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log - @itwin/insights-client
 
+## 0.5.4
+Tuesday, 12 December, 2023
+### Minor
```

Comment:
Minor change but only a patch bump?

Reply:
I always only check the message and miss the category of the change :|

Reply:
I see it's fixed in https://github.com/iTwin/insights-client/pull/48